globals_.py

```diff
@@ -3,7 +3,7 @@
 
 ACK_SIZE = 64 * 8  # 64 bytes
 DATA_PACKET_SIZE = 1024 * 8  # 1024 bytes
-LINK_MAX_BITS_IN_SEQUENCE = 10 ** 5
+LINK_MAX_BITS_IN_SEQUENCE = 10 ** 6
```

Comment:
This link-layer change made test case 0 go from about 50 seconds to about 27 seconds